nexus-repository-manager/README.md

```diff
@@ -16,6 +16,10 @@
 
 As of October 24, 2023, we will no longer update or support this Helm chart.
 
+Deploying Nexus Repository in containers with an embedded database has been known to corrupt the database under some circumstances. We strongly recommend that you use an external PostgreSQL database for Kubernetes deployments. If you are deploying in AWS, you can use our [AWS Helm chart](https://github.com/sonatype/nxrm3-helm-repository/tree/main/nxrm-aws-resiliency) to deploy Nexus Repository in an EKS cluster.
```

Comment:
Would it make sense to shorten this slightly?
> Deploying Nexus Repository in containers with an embedded database has been known to corrupt the database under some circumstances.

to

> Deploying Nexus Repository in containers  has been known to corrupt the embedded database under some circumstances.

Reply:
Oh wait, I take it back. We do need to specify that this only affects deployments that do use an embedded database, my shortened version obscures that.

Reply:
Would it make sense to somehow get in here that if you want to use a container, you should strongly consider the external database (available in these fine helm charts...) ?

Reply:
@mrprescott isn't that covered in the next few sentences?